### DISCUSSION TITLE
How to show all rows and columns ?

### DISCUSSION THREAD

**Discussion by madhavanhere** [duckdb/duckdb#8020] [Q&A] [answered]
I only see a subset of columns when I do a _Select * from table_
Is there any parameter I can add to show 'all columns'? 

Similarly with rows (show all rows for a small'ish table)?

**Comment by Alex-Monahan** [comment, marked as answer]
Hello! I assume you are using the CLI?

You can use `.maxrows 100000` to adjust the rows. If you want to display all data, you can use a different output mode (`.mode box`). There is not a .maxcol, but you can set the maxwidth to get columns to fit instead of `.mode box`.
https://github.com/duckdb/duckdb/discussions/5921

**Reply by madhavanhere** [reply]
Thanks Much!

**Comment by kaleidawave** [comment]
Is there a way to do this from the command (not the interactive CLI) aka `duckdb database.db -c "..."`. Want to bump the max output to 200.

**Reply by OhMyMndy** [reply]
You can use the flag `-box` so `duckdb -box database.db -c "select 1"`
